packages/frontend/src/app/(sticky-header)/all/[[...page]]/page.tsx

```diff
@@ -41,7 +42,7 @@ export default async function LatestPosts({
   const currentPage = !params.page ? 1 : Number(params.page?.[0])
 
   if (params.page?.length > 1 || !(currentPage > 0)) {
-    console.error('Incorrect page!', params.page, currentPage)
+    log(LogLevel.ERROR, `Incorrect page!: ${params.page}, ${currentPage}`)
```

Comment:
This seems to be a client error, could we use
`LogLevel.INFO` or `LogLevel.WARN` ?

Reply:
fixed.